manifests/init.pp

```diff
@@ -46,6 +46,14 @@
     },
   }
 
+  if fact('os.release.major') == '8' {
```

Comment:
It would probably be better to use a `>=` so it would also work with EL9.

Reply:
I specifically only wanted to do it on RHEL8. RHEL9 doesn't have "module streams", and the default mariadb-server package will do fine there.

Reply:
EL9 retains dnf modules (appstreams): https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/considerations_in_adopting_rhel_9/ref_application-streams_considerations-in-adopting-rhel-9

Reply:
Oh, odd. So on el9 mariadb is coming from the appstream repo but there is no associate dnf module to enable for different versions.  Maybe because 10.5 is the only option?

```
$ sudo dnf info mariadb
Last metadata expiration check: 2:14:18 ago on Mon 13 Mar 2023 09:15:03 PM UTC.
Available Packages
Name         : mariadb
Epoch        : 3
Version      : 10.5.16
Release      : 2.el9_0
Architecture : x86_64
Size         : 1.6 M
Source       : mariadb-10.5.16-2.el9_0.src.rpm
Repository   : appstream
Summary      : A very fast and robust SQL database server
URL          : http://mariadb.org
License      : GPLv2 and LGPLv2
Description  : MariaDB is a community developed fork from MySQL - a multi-user, multi-threaded
             : SQL database server. It is a client/server implementation consisting of
             : a server daemon (mariadbd) and many different client programs and libraries.
             : The base package contains the standard MariaDB/MySQL client programs and
             : utilities.

```

Reply:
Yes, I think that's correct. Both RHEL8 and RHEL9 have "AppStream" repos, but whereas in RHEL8 there are multiple versions of, eg mariadb, available that one can switch between as modules (default 10.3, 10.5 available); in RHEL9 there aren't different module versions ([yet](https://www.redhat.com/en/blog/modularity-red-hat-enterprise-linux-9), maybe there will be in future). The RHEL 9 default and only mariabd is 10.5.

To be honest, 10.3 would probably be fine for CCS, but it seemed like a good opportunity to start out with 10.5.